presets/mainnet/electra.yaml

```diff
@@ -38,3 +38,8 @@ MAX_CONSOLIDATIONS: 1
 MAX_DEPOSIT_RECEIPTS_PER_PAYLOAD: 8192
 # 2**4 (= 16) withdrawal requests
 MAX_WITHDRAWAL_REQUESTS_PER_PAYLOAD: 16
```

Comment:
Why don’t we re-use the already existing `MAX_WITHDRAWALS_PER_PAYLOAD? Is there any need in introducing a new parameter to limit a number of the same operation?

Reply:
We can also define `MAX_PENDING_PARTIALS_PER_WITHDRAWALS_SWEEP := MAX_WITHDRAWALS_REQUESTS_PER_PAYLOAD // 2` in the spec. (need some treatment in the specbuilder though)

Reply:
nvm, i’ve got confused